threedworld-mit/tdw
# Added ui.set_depth(id, depth)

How to test

-  Clone this repo
- Install TDW locally: cd tdw/Python ; git checkout set_ui_element_position ; pip3 install -e .
- Download and extract the test build: https://www.dropbox.com/scl/fi/trsbklgl7tc1dljv3k4jl/TDW_Windows_set_ui_element_depth.zip?rlkey=921x8nikqa8jadd35io9jydnm&dl=0

There is no preexisting test controller for setting a UI element's depth. You will need to write one yourself. See `lessons/ui/ui.md` under "Set a UI element's depth (z value)"

## Changelog

### Command API

#### New Commands

| Command | Description |
| --- | --- |
| `set_ui_element_depth` | Set the depth of a UI element. |

### `tdw` module

- Added: `ui.set_depth(id, depth)`.

### Documentation

#### Modified Documentation

| Document | Modification |
| --- | --- |
| `lessons/ui/ui.md` | Added a section describing how to set a UI element's depth. |
